client/components/site-selector/add-site.jsx

```diff
@@ -15,10 +15,8 @@ import PopoverMenu from 'components/popover/menu';
 import PopoverMenuItem from 'components/popover/menu-item';
 import { recordTracksEvent } from 'state/analytics/actions';
 import config from 'config';
-import sitesFactory from 'lib/sites-list';
 import { abtest } from 'lib/abtest';
-
-const sites = sitesFactory();
+import { hasJetpackSites } from 'state/selectors';
```

Comment:
👍  Good catch there are other cases that new selector could replace similar code, I will create a pull request with this kind of replacements and you will include this find in it :)